dashboard/app/models/plc/enrollment_unit_assignment.rb

```diff
@@ -23,6 +23,15 @@ class Plc::EnrollmentUnitAssignment < ActiveRecord::Base
   has_many :plc_module_assignments, class_name: '::Plc::EnrollmentModuleAssignment', foreign_key: 'plc_enrollment_unit_assignment_id', dependent: :destroy
   has_many :plc_task_assignments, through: :plc_module_assignments, class_name: '::Plc::EnrollmentTaskAssignment', dependent: :destroy
 
+  START_BLOCKED = 'start_blocked'
+  PENDING_EVALUATION = 'pending_evaluation'
+  IN_PROGRESS = 'in_progress'
+  COMPLETED = 'completed'
+
+  unit_status_states = [START_BLOCKED, PENDING_EVALUATION, IN_PROGRESS, COMPLETED]
```

Comment:
You can also do something like
```ruby
UNIT_STATUS_STATES = [
  START_BLOCKED = 'start_blocked'
  PENDING_EVALUATION = 'pending_evaluation'
  IN_PROGRESS = 'in_progress'
  COMPLETED = 'completed'
]
```
which will create all the constants and the container array at once.